src/app/legacy/containers/CpsRecommendations/RecommendationsPromoList/index.jsx

```diff
@@ -89,14 +68,12 @@ const RecommendationsPromoList = ({ promoItems }) => {
       enableGelGutters
```

Comment:
In the line above is `as={StoryPromoUl}`, which means this Grid will be a ul. In the Related Content code it looks possible for there to be a single related content item in the data. This might be rare, but do we need to allow for the condition where there is only one related content item, and in this case not have the item in a <ul>.?

Here is the related content section check for a single item https://github.com/bbc/simorgh/blob/7f770a1120275be841d3ada0adb15dafb4dfed24/src/app/components/RelatedContentSection/index.tsx#L187

Reply:
It's a good question. 

Curious why we have that logic in the related content component, because I can't see why it would be semantically incorrect to have a single list item in a list? 

A list can still be a list even if there's only 1 item in it. I'll double check the HTML reference.

Reply:
Suppose expecting in screen readers when it encounters a list may expect more than one item. 

I'll look into the complexities of having a similar approach to the related content component here, but may result to the "its an experiment" card if it messes the code up to much for now.

Reply:
Oh wait, the use-case for a single item is handled up a level: https://github.com/bbc/simorgh/blob/e32dcb04af40a335a75bdf8211ef1a2b8cb63c9e/src/app/legacy/containers/CpsRecommendations/index.jsx#L108-L112